presto-main/src/main/java/com/facebook/presto/sql/planner/RelationPlanner.java

```diff
@@ -151,14 +154,27 @@
         this.subqueryPlanner = new SubqueryPlanner(analysis, variableAllocator, idAllocator, lambdaDeclarationToVariableMap, metadata, session);
     }
 
+    private void checkMaxNodeNumInPlan(SqlPlannerContext context)
```

Comment:
We usually don't use abbreviations like num, and this also increments, not just check. So a better name would be something like: `incrementPlanLeaves`, or `incrementAndCheckPlanLeaves`

Reply:
This function makes more sense in `SqlPlannerContext` itself, and pass session as a variable.

Reply:
Done.